java/maven.embedder/src/org/netbeans/modules/maven/embedder/EmbedderFactory.java

```diff
@@ -56,6 +57,13 @@
  */
 public final class EmbedderFactory {
 
+    static{
+        // disable Maven's ANSI CLI decorations
+        // maven-slf4j-provider injects MavenSimpleLogger which decorates the output for terminal use
+        // this causes problems in log files, tests etc which don't expect or cant display them properly
+        MessageUtils.setColorEnabled(false);
+    }
```

Comment:
@sdedic is this a good place to put it? My thinking is that this is probably the main entry point into the lib wrapper module.